e2e/app/CreateAccountPage.ts

```diff
@@ -0,0 +1,218 @@
+import {ElementHandle} from 'puppeteer';
+import Button from './aou-elements/Button';
+import CheckBox from './aou-elements/CheckBox';
+import ClrIconLink from './aou-elements/ClrIconLink';
+import Label from './aou-elements/Label';
+import RadioButton from './aou-elements/RadioButton';
+import SelectComponent from './aou-elements/SelectComponent';
+import TextArea from './aou-elements/TextArea';
+import TextBox from './aou-elements/TextBox';
+import BasePage from './page-mixin/BasePage';
+
+import {defaultFieldValues} from '../resources/data/user-registration-fields';
+
+const faker = require('faker/locale/en_US');
+
+export const INSTITUTION_AFFILIATION = {
+  EARLY_CAREER_TENURE_TRACK_RESEARCHER: 'Early career tenure-track researcher',
+  UNDERGRADUATE_STUDENT: 'Undergraduate (Bachelor level) student',
+  INDUSTRY: 'Industry',
+};
+
+export const EDUCATION_LEVEL = {
+  DOCTORATE: 'Doctorate',
+};
+
+export const FIELD_LABEL = {
+  READ_UNDERSTAND_PRIVACY_STATEMENT: 'I have read and understand the All of Us Research Program Privacy Statement',
+  READ_UNDERSTAND_TERMS_OF_USE: 'I have read and understand the All of Us Research Program Terms of Use described above',
+  INSTITUTION_NAME: 'Institution Name',
+  ARE_YOU_AFFILIATED: 'Are you affiliated with an Academic Research Institution',
+  RESEARCH_BACKGROUND: 'Please describe your research background, experience and research interests',
+  EDUCATION_LEVEL: 'Highest Level of Education Completed',
+  YEAR_OF_BIRTH: 'Year of Birth',
+};
```

Comment:
best practice: externalize hardcoded strings outside of functions.
Same pattern are in many page objects.